apiRoutes/routes.js

```diff
@@ -0,0 +1,51 @@
+const router = require('express').Router();
+const axios = require('axios');
+
+const robokache = require('../src/API/robokache');
```

Comment:
I don't like the idea of sharing code between the frontend and backend. My view is that they are two distinct entities and are running in separate environments. Seems like we could run into weird errors if, for example, we tried to access `window` somewhere in the robokache source file. In addition, we have to share all dependenices between the frontend and backend which bloats the runtime environment of the backend and the bundle size of the frontend. It also just feels like bad practice to so tightly couple the components. 

We can't separate these now, but maybe we should think about trying not to share code so that we have more flexibility in the future. What that would look like is keeping everything in apiRoutes self contained. Right now that would involve copying the src/API/ara.js file into apiRoutes as well as duplicating some of src/API/robokache.js in apiRoutes.